chapters/intro-computer-architecture/binary-hex/reading/README.md

```diff
@@ -0,0 +1,142 @@
+# Binary and Hexadecimal Systems
+
+For representing information (instructions and data), computers use the binary system (base 2).
+When writing programs in assembly language, the hexadecimal system (base 16) is preferred because it saves the programmer from writing long strings of 1s and 0s, and conversion to/from binary can be done much more easily than with the decimal system (base 10).
+
+>**NOTE**: We'll use the prefix `0b` for representing numbers in binary and `0x` for hexadecimal numbers.
+> For example, we can write the unsigned integer `127` as `0b01111111` or `0x7F`.
+
+## Binary System
+
+In the **binary system** (base 2), values are represented as a string of 0s and 1s.
+Each digit in the string represents a bit, and a group of 8 bits forms a byte.
+A group of 4 bits is called a **nibble** or **half-byte**.
+
+### Operations with Values Represented in Binary
+
+#### Arithmetic Operations
+
+[Guide for Arithmetic Operations](https://www.tutorialspoint.com/computer_logical_organization/binary_arithmetic.htm)
```

Comment:
While I like how you used other website for additional information, I think the text could use some words explaining what the difference between arithmetic and logical operators, and a rough explanation on how the arithmetic operations in base 2 and base 10 differ. It doesn't need a full explanation, but something nonetheless.